semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/nodemethods/AstNodeMethods.scala

```diff
@@ -93,6 +94,15 @@ class AstNodeMethods(val node: AstNode) extends AnyVal with NodeExtension {
       case call: CallRepr if !call.isInstanceOf[Call] => call.code
     }
 
+  def sourceCode: String = {
+    val maybeSourceCode = for {
+      offset      <- node.offset
+      offsetEnd   <- node.offsetEnd
+      fileContent <- node.file.headOption.map(_.content)
+    } yield fileContent.substring(offset, offsetEnd)
+    maybeSourceCode.getOrElse(PropertyDefaults.Code)
```

Comment:
```suggestion
    maybeSourceCode.getOrElse(AstNode.PropertyDefaults.Code)
```
just my opinion, but I'd say it makes it more obvious that it's the AstNode specific default value...